x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -925,6 +1040,7 @@ const processRecoveredAlerts = ({
   spaceId: string;
   startedAt: Date;
   validatedParams: RuleParams;
+  getAlertByAlertUuid: (alertUuid: string) => { [x: string]: any } | null;
```

Comment:
Is there a type without `any` for the return value?

Reply:
Done